provisioner/roles/manage_ec2_instances/tasks/instances/instances_rhel.yml

```diff
@@ -130,3 +130,95 @@
   with_indexed_items:
     - "{{ node3_output.instances }}"
   when: node3_output.instance_ids is not none
+
+# Create isolated/remote node for Adv Tower lab when cluster install
+
+- name: Create EC2 instances for isonode
+  ec2:
+    assign_public_ip: true
+    key_name: "{{ ec2_name_prefix }}-key"
+    group: "{{ ec2_security_group }}"
+    instance_type: "{{ ec2_info[rhel].size }}"
+    image: "{{ node_ami.image_id }}"
+    region: "{{ ec2_region }}"
+    exact_count: "{{ student_total }}"
+    count_tag:
+      Workshop_isonode: "{{ ec2_name_prefix }}-isonode"
+    instance_tags:
+      Workshop_isonode: "{{ ec2_name_prefix }}-isonode"
+      Workshop: "{{ ec2_name_prefix }}"
+      Workshop_type: "{{ workshop_type }}"
+      AWS_USERNAME: "{{ aws_user }}"
+      Info: "AWS_USERNAME that provisioned this-> {{ aws_user }}"
+      Linklight: "This was provisioned through the linklight provisioner"
+      Students: "{{ student_total }}"
+      short_name: "isonode"
+      username: "{{ ec2_info[rhel].username }}"
+    wait: "{{ ec2_wait }}"
+    vpc_subnet_id: "{{ ec2_vpc_subnet_id }}"
+    volumes:
+      - device_name: /dev/sda1
+        volume_type: gp2
+        volume_size: "{{ ec2_info[rhel].disk_space }}"
+        delete_on_termination: true
+  register: isonode_output
+  when: create_cluster|bool
+
+- name: Ensure tags are present for isonode
+  ec2_tag:
+    region: "{{ ec2_region }}"
+    resource: "{{ item.1.id }}"
+    state: present
+    tags:
+      Name: "{{ ec2_name_prefix }}-student{{ item.0 + 1 }}-isonode"
+      Index: "{{ item[0] }}"
+      Student: "student{{ item.0 + 1 }}"
+      launch_time: "{{ item.1.launch_time }}"
+  with_indexed_items:
+    - "{{ isonode_output.instances }}"
+  when: isonode_output.instance_ids is not none
```

Comment:
@goetzrieger I think here you need to add a conditional `create_cluster|bool` as well.
Same further above. Otherwise it will try to assign tags to something which is not there.